source/standards/source-code.html.md.erb

```diff
@@ -80,6 +80,8 @@ Ensure the repository permissions follow [the principle of least privilege](/sta
 If your repository has external contributors, [ensure they do not have permissions to add workflows or trigger workflow runs](https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks).
 
 If your workflow interacts with another resource (for example AWS or DockerHub), consider setting up a dedicated account or role, with permissions limited to the scope of the action.
+Use temporary credentials in preference to storing long-lived credentials in a secret.
+In particular, when accessing AWS you must [authenticate using the OpenID Connect token][GHA-AWS-OIDC] and not using an IAM User's access key and secret access key.
```

Comment:
Add "You should also specify the branch you expect to be deploying from (e.g. "main") in your IAM role to make sure code cannot be deployed from untrusted branches."

Reply:
raised https://github.com/alphagov/gds-way/pull/667 for this